packages/@aws-cdk/aws-securityhub/lib/hub.ts

```diff
@@ -0,0 +1,152 @@
+import events = require('@aws-cdk/aws-events');
+import cdk = require('@aws-cdk/core');
+import securityhub = require('./securityhub.generated');
+
+/**
+ * Properties for SecurityHub Hub.
+ */
+export interface HubProps {
+    /**
+     * A key-value map of resource tags of this Hub.
+     *
+     * @default - Empty tags
+     */
+    readonly tags?: { [key: string]: string };
```

Comment:
Do tags not work out-of-the box using the aspect mechanism?

Reply:
Yes, I think you are right.